wagtail/tests/urls.py

```diff
@@ -42,6 +44,16 @@
     path("fallback/", lambda: HttpResponse("ok"), name="fallback"),
 ]
 
+if apps.is_installed("pattern_library"):
+    urlpatterns += [
+        path(
+            "pattern-library/api/v1/sprite",
```

Comment:
Should this URL be also pulled out into a shared const OR a window variable, similar to `window.PATTERN_LIBRARY_API`?

Reply:
Yep, I’ve done something similar. I think it’ll only be used in one place so have kept it relatively simple.